internal/assets/migrations/002_tx_info.sql

```diff
@@ -0,0 +1,17 @@
+-- +migrate Up
+CREATE TYPE tx_status_enum AS ENUM ('pending', 'completed', 'failed');
+
+ALTER TABLE participants
+    ADD COLUMN status     tx_status_enum              NOT NULL DEFAULT 'completed',
+    ADD COLUMN updated_at TIMESTAMP WITHOUT TIME ZONE NOT NULL DEFAULT NOW(),
+    ADD COLUMN tx_hash    VARCHAR(64)                 NOT NULL DEFAULT '',
+    ADD COLUMN amount     TEXT                        NOT NULL DEFAULT '0urmo';
+
+-- +migrate Down
+ALTER TABLE participants
```

Comment:
Yes, it will, but this statements are for cases to rollback for specific "version" of the database, it seems to me that it's a good practice to have such an option.